tests/cases/fourslash/getOutliningForObjectsInArray.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference path="fourslash.ts"/>
+
+////// objects in x should generate outlining spans that do not render in VS

```

Comment:
Unless you want the comment to be part of the sourcefile, you can just use two slashes.